### PR TITLE
Added GetEntries

### PR DESCRIPTION
**Pull request by spearmin10**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
A wrapper script of `getEntries` which is a build-in function. This collects entries from the war room with conditions given.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by spearmin10** [comment]
> Hi @spearmin10, thank you for your contribution. Can you please elaborate, what is the purpose of this script? what are the use cases? Also, what do you mean by incident id as an argument, I encountered the attached problem when trying to execute the script with an existing incident I have in my localhost. Screen Shot 2022-09-14 at 18 05 09

**Comment by spearmin10** [comment]
@YuvHayun How did you import the script? I'm not sure why the error raises because The line 32 is below. I think that the script imported is something different.
image

**Comment by spearmin10** [comment]
@YuvHayun It's a simply wrapper of buildin `getEntries`, so there are various use cases. In my use case, I wanted to collect tagged entries with specific tags to extract indicators from them.

**Comment by YuvHayun** [comment]
> @YuvHayun How did you import the script? I'm not sure why the error raises because The line 32 is below. I think that the script imported is something different. image

Not sure why it's related to the imports, I ran the same command the same way twice with two different incident ids that I have both in my incidents tab - one failed and one didn't.

**Comment by spearmin10** [comment]
@YuvHayun I've fixed the issue. Please review again.

File: Packs/CommonScripts/Scripts/GetEntries/GetEntries_test.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+from CommonServerPython import *
```
Comment (YuvHayun) [inline Packs/CommonScripts/Scripts/GetEntries/GetEntries_test.py:1]: Let's add "given-when-then" documentation to this tests cases.

**Review by YuvHayun — CHANGES_REQUESTED** [review]
Hey @spearmin10, thx for the fixes.
I added 1 minor comment.
Also, I still get the same failures when running the same command as I showed you on our call last Thursday.
Also, I think there's something inefficient in the way we print so many msgs to the war room.
Please consider changing the way the human-readable is printed to the war-room.

File: Packs/CommonScripts/Scripts/GetEntries/GetEntries_test.py (hunk 1 above)
Comment (spearmin10) [reply Packs/CommonScripts/Scripts/GetEntries/GetEntries_test.py:1]: @YuvHayun Could you paste the screenshot of what the error raised?

**Review by YuvHayun — APPROVED** [review]

**Comment by YuvHayun** [comment]
@spearmin10 merging this pr.
Thx for your contribution!

**Pull request merged**